lib/manager/components/deployment/DeploymentViewer.js

```diff
@@ -76,6 +76,10 @@ export default class DeploymentViewer extends Component<Props, State> {
   }
 
   componentDidMount () {
+    const { deployment, fetchDeployment } = this.props
+    // If there is no project id, we are dealing with a summary and need to fetch the
```

Comment:
Should there be a check here to see if there is no project id?

Reply:
Yep thanks for catching this! This comment was outdated.